autoload/neomake/makers/ft/erlang/gradualizer.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+#set -x
+
+# Set to your gradualizer location or make sure the command is in PATH
+GRADUALIZER=gradualizer
+
+if [ x`uname` = x"Darwin" ]; then
+    SED=gsed
+else
+    SED=sed
+fi
```

Comment:
This doesn't look that good: it shells out to `uname` for once.
Is GNU sed really required?
And shouldn't the errorformat just handle this (the replacement) maybe even?

Reply:
GNU and BSD sed do behave differently with the present command and I was not able to come up with a sed script/command which would achieve the same but work for both of them.
Then, it's arguable, but I think more probable that a developer using a Mac will have gsed installed, than a Linux person a BSD sed installed.

I'll look into `errorformat`, maybe you're right that it's possible to ditch the shell script altogether.